measurements/attribute_based.py

```diff
@@ -74,59 +74,60 @@ def att_siz(dataloader, args):
         if len(attribute)> 1:
             shape = list(data.size())[1:]
             for index in range(len(attribute[1])):
-                bbox = attribute[1][index]
-                att = attribute[0][index]
-                bbox_adjust = np.array([bbox[0]*shape[1], bbox[1]*shape[1], bbox[2]*shape[0], bbox[3]*shape[0]])
-                pixel_size = (bbox_adjust[1]-bbox_adjust[0])*(bbox_adjust[3]-bbox_adjust[2])
-                size = (bbox[1]-bbox[0])*(bbox[3]-bbox[2])
-                person_center = np.array([bbox[0] + (bbox[1]/2.), bbox[2] + (bbox[3]/2.)])
-                distance = np.linalg.norm(person_center - img_center)
-                pic = data.data.cpu().numpy()
+                if type(attribute[1][index])== list:
```

Comment:
rather than checking if it's a list, maybe check it is not None or something, since it could be an array